### PR TITLE
add debug info in scaling cluster with WLDF methods

### PR DESCRIPTION
**Pull request by xiancao**
Add more debug information in scaling cluster with WLDF methods.

https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/1731/
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/1732/
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/1745/

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java [hunk 1]
```diff
@@ -396,6 +398,10 @@ public static boolean scaleClusterWithWLDF(String clusterName,
     ExecResult result = exec(adminPod, null, true,
         "/bin/sh", "-c", "mkdir -p " + domainHomeLocation + "/bin/scripts");
     if (result.exitValue() != 0) {
+      logger.info("failed to create directory {0}/bin/scripts on admin server pod", domainHomeLocation);
+      logger.info("result.existValue={0}", result.exitValue());
```
Comment (vanajamukkara) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java:402]: exitValue

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java [hunk 2]
```diff
@@ -405,17 +411,26 @@ public static boolean scaleClusterWithWLDF(String clusterName,
...
+          PROJECT_ROOT + "/../src/scripts/scaling/scalingAction.sh", adminServerPodName, domainNamespace,
+          ioex.getStackTrace());
       return false;
     }
 
     logger.info("Adding execute mode for scalingAction.sh");
     result = exec(adminPod, null, true,
         "/bin/sh", "-c", "chmod +x " + domainHomeLocation + "/bin/scripts/scalingAction.sh");
     if (result.exitValue() != 0) {
+      logger.info("failed to add execute mode for file {0} in admin server pod {1}",
```
Comment (vanajamukkara) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java:429]: modify Kubernetes.exec() method to log the msg, stdout and stderr when exitValue is non-zero, then you don't have to log at the caller

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java (hunk 1 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java (hunk 2 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java:426]: fixed

**Pull request merged**